### DISCUSSION TITLE
Why do breakpoints use "em" isntead of "rem" or some other unit?

### DISCUSSION THREAD

**Discussion by zenVentzi** [chakra-ui/chakra-ui#4334] [Q&A]
Out of curiosity, just trying to learn. Why "em" has been considered to be the optimal choice instead of rem or some other unit?

UPDATE: posted the question without reading first. My apologies. Couldn't find a way to delete the discussion though..

**Comment by benknab** [comment]
@zenVentzi, I think it's a fair question; I would love to hear what answer you found. 
Tailwind uses `px` instead of `em`, so I don't think it's trivial.
